packages/odata-common/src/request/odata-function-import-request-config.ts

```diff
@@ -4,7 +4,7 @@ import { FunctionImportParameters } from './function-import-parameter';
 import { ODataRequestConfig, RequestMethodType } from './odata-request-config';
 
 /**
- * @internal
+ * PLEASE REVIEW.
```

Comment:
Just copy the description from the other request configs like ODataCountRequestConfig and adjust the @typeparam if needed.